envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadataTest.kt

```diff
@@ -390,7 +554,28 @@ class NodeMetadataTest {
             )
         }
         assertThat(exception.status.description)
-                .isEqualTo("Invalid access log status code filter. Expected OPERATOR:STATUS_CODE")
+            .isEqualTo("Invalid access log status code filter. Expected OPERATOR:STATUS_CODE")
         assertThat(exception.status.code).isEqualTo(Status.Code.INVALID_ARGUMENT)
     }
+
+    @Suppress("UnnecessaryApply")
```

Comment:
Why is this suppressed?

Reply:
I want to return "ObjectAssert<DependencySettings>" type but klint/detekt doesn't see my intension

Reply:
I refactored it and remoded Suppress